src/rust/ide/view/graph-editor/src/component/node.rs

```diff
@@ -12,12 +12,15 @@ pub mod output;
 pub mod error;
 #[deny(missing_docs)]
 pub mod vcs;
+mod profiling_indicator;
 
 pub use error::Error;
 pub use expression::Expression;
 
 use crate::prelude::*;
 
+use crate::component::node::profiling_indicator::ProfilingIndicator;
+use crate::Mode as EditorMode;
```

Comment:
The name `display` is already taken. Do you have another name that you would be happy with?

Reply:
We are currently having a call about it and are thinking about the name `view::Mode`.

Reply:
Did you have any new thoughts on this? @wdanilo

Reply:
If we plan to only use this only for the profiling mode then it could make sense to move it to `profiling::Mode::On` and `profiling::Mode::Off`. Otherwise, I would suggest `display_mode::DisplayMode::Normal` and `display_mode::DisplayMode::Profiling`. Or just leaving it where it is and renaming `Mode` to `DisplayMode`.

Reply:
I'd rather give us possibility to implement easily other modes without major refacotirng so I'd not go with `profiling::Mode::On`. On the other hand, the naming `display_mode::DisplayMode` is what we try not to do, as it is repeating itself and preventing nice qualified usage. I think that `crate::View::Mode::Normal` is much better (as we discussed on the call).